docker/Dockerfile

```
@@ -1,5 +1,8 @@
 FROM python:3.11.5
 
+# Set a default value for AP_VERSIONS
+ENV AP_VERSIONS='{"AP45": "0.12.27139", "AP32": "0.12.27139"}'
```

If the user does not provide AP_VERSIONS, id rather it fail and asked the user to provide it rather than do this by default.